### PR TITLE
Expand usable ingredients in cooking recipes

### PR DESCRIPTION
**Pull request by Rail-Runner**
#### Summary

SUMMARY: Content "Expand usable ingredients in cooking recipes"
#### Purpose of change

Add already existing ingredients to more recipes, mostly ingredients that are obtained from butchering mutated creatures and humanoids, and add several ingredient lists to ensure recipe consistency.
#### Describe the solution

JSON editing.
#### Describe alternatives you've considered

None.
#### Testing

Loaded the game, made sure that no errors happen.
#### Additional context

None.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
If you're up for it, it looks like a lot of these should be requirements, not individual lists.

**Comment by Rail-Runner** [comment]
> If you're up for it, it looks like a lot of these should be requirements, not individual lists.

Which ones, specifically? I considered that it's only worth it for canning or similar recipe which use exactly the same ingredients, but I decided to not significantly change canning recipes for now.

File: data/json/recipes/armor/pets_horse.json [hunk 1]
```diff
@@ -77,6 +77,8 @@
         [ "wax", 6 ],
         [ "tallow", 24 ],
         [ "mutant_tallow", 24 ],
+        [ "human_tallow", 24 ],
+        [ "demihuman_tallow", 24 ],
```
Comment (anothersimulacrum) [inline data/json/recipes/armor/pets_horse.json:81]: Some sort of fats requirement.

File: data/json/recipes/chem/mutagens.json [hunk 2]
```diff
@@ -269,7 +269,7 @@
     "batch_time_factors": [ 80, 20 ],
     "book_learn": [ [ "recipe_animal", 5 ] ],
     "using": [ [ "mutagen_production_standard", 25 ] ],
-    "components": [ [ [ "mutagen", 1 ] ], [ [ "meat", 3 ] ], [ [ "ammonia", 1 ], [ "lye_powder", 100 ] ] ],
+    "components": [ [ [ "mutagen", 1 ] ], [ [ "meat", 3 ], [ "meat_scrap", 30 ] ], [ [ "ammonia", 1 ], [ "lye_powder", 100 ] ] ],
```
Comment (anothersimulacrum) [inline data/json/recipes/chem/mutagens.json:272]: meat_or_scrap

File: data/json/recipes/food/baking.json [hunk 3]
```diff
@@ -15,9 +15,9 @@
     "components": [
       [ [ "flour", 4 ] ],
       [ [ "yeast", 2 ] ],
-      [ [ "apple_cider", 2 ], [ "apple", 2 ] ],
-      [ [ "pumpkin", 2 ] ],
-      [ [ "sugar", 4 ] ],
+      [ [ "apple_cider", 2 ], [ "apple", 2 ], [ "irradiated_apple", 2 ] ],
+      [ [ "pumpkin", 2 ], [ "irradiated_pumpkin", 2 ] ],
+      [ [ "sugar", 4 ], [ "artificial_sweetener", 4 ] ],
```
Comment (anothersimulacrum) [inline data/json/recipes/food/baking.json:20]: sweeteners_any

File: data/json/recipes/food/offal_dishes.json [hunk 4]
```diff
@@ -307,15 +307,15 @@
...
+      [ [ "meat_lung", 4, "LIST" ] ],
       [ [ "flour", 4 ] ],
       [ [ "onion", 1 ], [ "irradiated_onion", 1 ] ],
       [ [ "cooking_oil", 1 ], [ "cooking_oil2", 1 ] ],
       [ [ "salt", 6 ] ],
       [ [ "pepper", 6 ] ],
       [ [ "worthy_wine", 3, "LIST" ] ],
       [ [ "garlic_clove", 1 ] ],
-      [ [ "sauce_pesto", 1 ], [ "sauce_red", 1 ], [ "tomato", 1 ] ]
+      [ [ "sauce_pesto", 1 ], [ "sauce_red", 1 ], [ "tomato", 1 ], [ "irradiated_tomato", 1 ] ]
```
Comment (anothersimulacrum) [inline data/json/recipes/food/offal_dishes.json:318]: pasta_sauce

File: data/json/recipes/recipe_others.json [hunk 5]
```diff
@@ -274,7 +274,7 @@
     "batch_time_factors": [ 83, 3 ],
     "flags": [ "BLIND_EASY" ],
     "tools": [ [ [ "rock_quern", -1 ], [ "clay_quern", -1 ] ] ],
-    "components": [ [ [ "chitin_piece", 1 ], [ "acidchitin_piece", 1 ] ] ]
+    "components": [ [ [ "chitin_piece", 1 ], [ "acidchitin_piece", 1 ], [ "endochitin", 1 ] ] ]
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_others.json:277]: chitins

**Pull request merged**